src/video-iframe-integration.js

```diff
@@ -207,15 +208,17 @@ export class AmpVideoIntegration {
   listenTo(type, obj, opt_initializer) {
     userAssert(
       !this.usedListenToHelper_,
-      '%s `listenTo` is meant to be used once per page.',
-      TAG
+      '`listenTo` is meant to be used once per page.'
     );
     const types = {
       'jwplayer': () => {
+        userAssert(
+          !obj.tagName,
```

Comment:
For this check, let's use a property that we need later? (`on`, `setMute`, etc.)